packages/vscode-extension-serverless-workflow-editor/it-tests/serverless-workflow-editor-extension-syntax-highlight.test.ts

```diff
@@ -0,0 +1,92 @@
+
+  let testHelper: VSCodeTestHelper;
+  let driver: WebDriver;
+
+  before(async function () {
+    this.timeout(30000);
+    testHelper = new VSCodeTestHelper();
+    await testHelper.openFolder(TEST_PROJECT_FOLDER, "syntax-highlight");
+  });
+
+  beforeEach(async function () {
+    await testHelper.closeAllEditors();
+    await testHelper.closeAllNotifications();
+  });
+
+  afterEach(async function () {
+    this.timeout(15000);
+    await testHelper.closeAllEditors();
+    await testHelper.closeAllNotifications();
+  });
+
+  it("Checks syntax highlighting of *.sw.json files in serverless workflow editor", async function () {
+    this.timeout(30000);
+
+    const editorWebviews = await testHelper.openFileFromSidebar("syntax-highlight-hello-world.sw.json");
+    const swfTextEditor = new SwfTextEditorTestHelper(editorWebviews[0]);
+    const textEditor = await swfTextEditor.getSwfTextEditor();
+
+    driver = textEditor.getDriver();
+
+    // get color codes of json properties
+    const property1WebElement = await getWebElementByDisplayedText("specVersion");
+    const property2WebElement = await getWebElementByDisplayedText("states");
+    const property1Color = await resolveWebElementColor(property1WebElement);
+    const property2Color = await resolveWebElementColor(property2WebElement);
+
+    // get color codes of json string values
+    const stringValue1WebElement = await getWebElementByDisplayedText("helloworld");
+    const stringValue2WebElement = await getWebElementByDisplayedText("HelloWorldWorkflow");
+    const stringValue1Color = await resolveWebElementColor(stringValue1WebElement);
+    const stringValue2Color = await resolveWebElementColor(stringValue2WebElement);
+
+    // get color code of json boolean value
+    const booleanValueWebElement = await getWebElementByDisplayedText("true");
+    const booleanValueColor = await resolveWebElementColor(booleanValueWebElement);
+
+    // check that properties have the same color
+    expect(property1Color == property2Color, "All properties should be the same color").to.be.true;
+
+    // check that string values have the same color
+    expect(stringValue1Color == stringValue2Color, "All string values should be the same color").to.be.true;
+
+    // check that properties, string values and boolean have different color
+    const allTypesColors = [property1Color, stringValue1Color, booleanValueColor];
+    const uniqueColors = Array.from(new Set(allTypesColors));
+    expect(allTypesColors.length, "Colors of properties, string values and boolean values should differ").equal(
+      uniqueColors.length
+    );
+  });
+
+  async function getWebElementByDisplayedText(displayedText: string): Promise<WebElement> {
+    return await driver.findElement(By.xpath("//*[text()[contains(.,'" + displayedText + "')]]"));
+  }
+
+  async function resolveWebElementColor(webElement: WebElement): Promise<string> {
+    return await (await Promise.resolve(webElement)).getAttribute("class");
```

Comment:
I would say this is not necessary.

```suggestion
    return await webElement.getAttribute("class");
```